### PR TITLE
chore: update testing-library, husky and lerna major versions

### PR DESCRIPTION
**Pull request by balzss**
INSTUI-3847
INSTUI-3936

### CODE REVIEW

**Review by matyasf — CHANGES_REQUESTED** [review]
CI is failing with 
`@instructure/ui-file-drop:   350:9   error    A form label must be associated with a control       jsx-a11y/label-has-associated-control`

**Comment by balzss** [comment]
> CI is failing with `@instructure/ui-file-drop: 350:9 error A form label must be associated with a control jsx-a11y/label-has-associated-control`

yeah, it's tricky because if I fix it and commit it, our linter changes it back which throws the lint error... I'll find another way I guess

File: packages/ui-source-code-editor/package.json [hunk 1]
```diff
@@ -26,24 +26,24 @@
...
-    "@codemirror/search": "^6.5.4",
-    "@codemirror/state": "^6.3.0",
-    "@codemirror/view": "^6.21.3",
+    "@codemirror/lang-markdown": "^6.2.5",
+    "@codemirror/language": "^6.10.1",
+    "@codemirror/legacy-modes": "^6.4.0",
+    "@codemirror/lint": "^6.7.1",
+    "@codemirror/search": "^6.5.6",
+    "@codemirror/state": "^6.4.1",
+    "@codemirror/view": "^6.26.3",
```
Comment (matyasf) [inline packages/ui-source-code-editor/package.json:46]: This went trough without bugs this time?

**Review by matyasf — APPROVED** [review]

File: packages/ui-source-code-editor/package.json (hunk 1 above)
Comment (balzss) [reply packages/ui-source-code-editor/package.json:46]: yes and I didn't need to pin any packages this time. I'm not sure why it was failing previous times

**Review by HerrTopi — APPROVED** [review]

**Pull request merged**